### PR TITLE
Add new menu to draw a mesh

### PR DESCRIPTION
**Pull request by YenPeiChen07**
This PR is related with #276

There are 3 changes: 
1. created a new menu called `Mesh` to  collect the mesh example. Currently, it only has one item `load_mh_3dmix`. 
2. enlarge sub-window size to make it view easier.
3. change the camera position to have a good view initially.
 
When selecting the menu item, the mesh will be drawn in a sub-window in the viewer.

- Select `load_mh_3dmix` then draw the mesh in sub-window. 
Screenshot 2024-03-04 at 9 21 16 AM

- Load the mesh again. Manually adjust right sub-window size to check out the result.
Screenshot 2024-03-04 at 9 21 43 AM

### CODE REVIEW

File: cpp/modmesh/view/R3DWidget.cpp [hunk 1]
```diff
@@ -44,7 +44,7 @@ R3DWidget::R3DWidget(Qt3DExtras::Qt3DWindow * window, RScene * scene, QWidget *
     // Set up the camera.
     Qt3DRender::QCamera * camera = m_view->camera();
     camera->lens()->setPerspectiveProjection(45.0f, 16.0f / 9.0f, 0.1f, 1000.0f);
-    camera->setPosition(QVector3D(0.0f, 0.0f, 10.0f));
+    camera->setPosition(QVector3D(1.0f, 2.0f, 8.0f));
```
Comment (yungyuc) [inline cpp/modmesh/view/R3DWidget.cpp]: This changes the initial camera direction for all apps.  Don't do this global change at this point.  Please revert this or make it configurable from Python and set it from the new menu item.

File: cpp/modmesh/view/RManager.cpp [hunk 2]
```diff
@@ -91,7 +91,7 @@ R3DWidget * RManager::add3DWidget()
         viewer->setWindowTitle("3D viewer");
         viewer->show();
         auto * subwin = this->addSubWindow(viewer);
-        subwin->resize(300, 200);
+        subwin->resize(400, 300);
```
Comment (yungyuc) [inline cpp/modmesh/view/RManager.cpp:94]: This makes sense.

File: modmesh/app/mh_3dmix.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+
+"""
+Show sample mesh
```
Comment (yungyuc) [inline modmesh/app/mh_3dmix.py]: Please add more description in the module docstring.

File: modmesh/app/mh_3dmix.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+"""
+
+from .. import core
+from .. import view
+from .. import apputil
+
+
+def help_3dmix(set_command=False):
```
Comment (yungyuc) [inline modmesh/app/mh_3dmix.py]: This function is not called.  Are you sure it's needed?

**Review by yungyuc — CHANGES_REQUESTED** [review]
Please make some changes:
- [x] Move the "Mesh" menu to be right after "File".
- [x] Avoid changing the initial camera direction globally.
- [x] If `help_3dmix` is not required, remove it.
- [x] Please make sure all CI tasks are good before requesting next review (Python linter fails: https://github.com/solvcon/modmesh/actions/runs/8135138740/job/22239175666?pr=282#step:11:32)

You can use your fork to run the CI by turning it on in the setting:

image

**Comment by YenPeiChen07** [comment]
@yungyuc, I have updated the code for the requested changes. 

Currently, "Mesh" menu is right after "File" and revert the camera position to original one. Additionally, the console will display the number of edges of the mesh when drawing the mesh.

Screenshot 2024-03-08 at 1 49 46 PM

**Review by yungyuc — APPROVED** [review]
LGTM

**Pull request merged**